elixir-format.el

```diff
@@ -125,54 +125,76 @@ Shamelessly stolen from go-mode (https://github.com/dominikh/go-mode.el)"
               (error "Invalid rcs patch or internal error in elixir-format--apply-rcs-patch"))))))))
   )
 
+(defun elixir-mix-path-set? ()
+  (and (not (= (length elixir-format-mix-path) 0)) (not (= (length elixir-format-elixir-path) 0)))
+  )
+
+(defun display-configuration-missing-error ()
+  (let ((errbuff (get-buffer-create "*elixir-format-configuration-missing-errors*")))
+    (progn
+      (with-current-buffer errbuff
+        (setq buffer-read-only nil)
+        (erase-buffer)
+        (insert "Please check https://github.com/elixir-editors/emacs-elixir#elixir-format docs to customize necessary variables for elixir and mix.")
```

Comment:
It's unpleasant to have an error in emacs force someone to go online to check the docs. Emacs should be self-documenting. It would be better to add a better docstring to the two variables (for instance like https://github.com/emacs-mirror/emacs/blob/7ce25de888f285502898da13e689e4372fc6a49d/lisp/eshell/em-cmpl.el#L97 ) that explains how to set the variables -- once that's done, this error can just ask people to refer to the documentation for the proper variables.

Reply:
done.